### ISSUE TITLE
[1.16.1] Redstone ore texture appears as regular stone

### ISSUE THREAD

**Issue by MarkKoz** [CyanideX/Unity-Extras#4]
**Describe the Issue**
Redstone ore texture appears identical to regular stone. The red particles are still emitted. The texture looks correct when the ore block is in the inventory.

**To Reproduce**
1. Install OptiFine
2. Install and enable the Unity and Unity Extras resources packs
3. Probably use the settings files I attached below
4. Load a world and place a redstone ore block
5. Observe the incorrect texture

**Expected Behavior**
The texture appears similar to the regular redstone ore texture i.e. 

![ore2](https://user-images.githubusercontent.com/1515135/89226116-405c0200-d590-11ea-89f5-74cf35ded006.png)

**Screenshots**
![ore](https://user-images.githubusercontent.com/1515135/89225662-6df47b80-d58f-11ea-9fed-90db0cc7a506.png)

**Additional Context**
Add any other context about the problem here. Do you happen to have any mods installed? Do you have any other resource packs enabled? Are you using OptiFine?

Only mod is OptiFine. Forge is not installed.

```
[14:02:32] [main/INFO]: [OptiFine] 
[14:02:32] [main/INFO]: [OptiFine] OptiFine_1.16.1_HD_U_G2_pre13
[14:02:32] [main/INFO]: [OptiFine] Build: 20200730-193926
[14:02:32] [main/INFO]: [OptiFine] OS: Windows 10 (amd64) version 10.0
[14:02:32] [main/INFO]: [OptiFine] Java: 14, Oracle Corporation
[14:02:32] [main/INFO]: [OptiFine] VM: OpenJDK 64-Bit Server VM (mixed mode, sharing), Oracle Corporation
[14:02:32] [main/INFO]: [OptiFine] LWJGL: 3.3.0 Win32 WGL EGL OSMesa VisualC DLL
[14:02:32] [main/INFO]: [OptiFine] OpenGL: GeForce GTX 980 Ti/PCIe/SSE2, version 4.6.0 NVIDIA 442.59, NVIDIA Corporation
[14:02:32] [main/INFO]: [OptiFine] OpenGL Version: 4.6.0
[14:02:32] [main/INFO]: [OptiFine] Maximum texture size: 16384x16384
[14:02:32] [main/INFO]: [Shaders] OpenGL Version: 4.6.0 NVIDIA 442.59
[14:02:32] [main/INFO]: [Shaders] Vendor:  NVIDIA Corporation
[14:02:32] [main/INFO]: [Shaders] Renderer: GeForce GTX 980 Ti/PCIe/SSE2
[14:02:32] [main/INFO]: [Shaders] Capabilities:  2.0  2.1  3.0  3.2  4.0 
[14:02:32] [main/INFO]: [Shaders] GL_MAX_DRAW_BUFFERS: 8
[14:02:32] [main/INFO]: [Shaders] GL_MAX_COLOR_ATTACHMENTS_EXT: 8
[14:02:32] [main/INFO]: [Shaders] GL_MAX_TEXTURE_IMAGE_UNITS: 32
```

[options.txt](https://github.com/Unity-Resource-Pack/Unity-Extras/files/5018448/options.txt)
[optionsof.txt](https://github.com/Unity-Resource-Pack/Unity-Extras/files/5018449/optionsof.txt)

**Comment by CyanideX** [comment]
Fixed in d245b0b 🙂

**Comment by CyanideX** [comment]
But actually fixed in e03a701 😅
